cmd/main.go

```diff
@@ -48,14 +45,18 @@ func run() error {
 func init() {
 	// for storage and service discovery
 	flags.AddConsulFlags(rootCmd, &opts)
+	// always use consul for storage and service discovery
+	rc.Options = opts
```

Comment:
init functions are called before the main function in go, and i think (though not sure) that the opts struct is initialized in main (in rootCmd.Execute). 

so i think it will be best to move this to a function called after main. if you need it more than once perhaps extract to a function?

Reply:
yes, that part was making me uncomfortable. AddConsulFlags configures the options then we overlay some additional properties. Maybe those properties should be set in `AddConsulFlagsForGlooConnect`?
For now, I moved it into a local function to encapsulate the config initialization